### ISSUE TITLE
Docker - ImportError: No module named pathlib2

### ISSUE THREAD

**Issue by Techbrunch** [Gallopsled/pwntools#2119]
I ran into the same issue mentioned in #2075, the Docker image appears to be broken:

```
pwntools@68f2300f39e0:/tst$ python fuzz.py
Traceback (most recent call last):
  File "fuzz.py", line 1, in <module>
    from pwn import *
  File "/usr/local/lib/python2.7/dist-packages/pwn/__init__.py", line 4, in <module>
    from pwn.toplevel import *
  File "/usr/local/lib/python2.7/dist-packages/pwn/toplevel.py", line 36, in <module>
    from pwnlib.filesystem import *
  File "/usr/local/lib/python2.7/dist-packages/pwnlib/filesystem/__init__.py", line 5, in <module>
    from pwnlib.filesystem.path import Path
  File "/usr/local/lib/python2.7/dist-packages/pwnlib/filesystem/path.py", line 7, in <module>
    from pathlib2 import *
ImportError: No module named pathlib2
```

**Comment by peace-maker** [comment]
This issue only appears for Python 2.7 and was fixed in #2179 by adding `pathlib2; python_version < '3.4'` to the install dependencies. That change isn't in the stable branch yet though, ~~which we should fix before pushing new docker images in #2204.~~

**Comment by peace-maker** [comment]
The docker images are up-to-date again.
